bosh-setup/nestedtemplates/storage-account-v1.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {
+    "storageAccountName": "[concat(uniqueString(resourceGroup().id), 'cfdefaultsa')]",
+    "storageAccountType": "Standard_LRS",
+    "storageAccountId": "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "sku": {
+        "name": "[variables('storageAccountType')]"
+      },
+      "kind": "Storage",
+      "name": "[variables('storageAccountName')]",
+      "apiVersion": "2016-01-01",
+      "location": "[resourceGroup().location]"
+    }
+  ],
+  "outputs": {
+    "storageAccountName": {
+      "type": "string",
+      "value": "[variables('storageAccountName')]"
+    },
+    "storageAccountKey": {
+      "type": "string",
+      "value": "[listKeys(variables('storageAccountId'), '2015-06-15').key1]"
```

Comment:
The current apiVersion works well for us. I don't have a particular reason to update the api version. Could you please share the benefit of updating apiVersion?

Reply:
BP for the repo is to use apiVersions that are no more than 1 year old - we block at 2 years

Reply:
Thanks @bmoore-msft .
I checked the doc again.
https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-profiles-azure-resource-manager-versions
The API versions for Azure stack are pretty old.
Is it possible to change BP a litter bit for the AzureStack-compatible template? I prefer not updating the versions.

Reply:
ok for now - AzureStack has it's own QS repo because it's so far behind in many places... eventually we'll automate this check, so something to keep in mind for the future (and we'll probably flag it next time too so feel free to remind me)